features/nanostack/mbed-mesh-api/source/include/enet_tasklet.h

```diff
@@ -22,10 +22,42 @@
 extern "C" {
 #endif
 
+/*
+ * \brief Initialize system.
+ */
 void enet_tasklet_init(void);
-uint8_t enet_tasklet_network_init(int8_t);
+
+/*
```

Comment:
These don't have the correct `/**` Doxygen markers - but if you do add them, do also add `\internal` commands, as I nothing here is public API .

Reply:
(Actually, you can probably put some sort of big group marker that says the whole thing is internal. Not sure of exact syntax)

Reply:
Actually the other tasklet header files(thread_tasklet.h, nd_tasklet.h etc) are documented same way than enet_tasklet now

Reply:
Looks like a new tracking issue to use doxygen but having it internal